homeassistant/components/hydrawise/config_flow.py

```diff
@@ -87,3 +94,39 @@ async def async_step_reauth(
     ) -> ConfigFlowResult:
         """Perform reauth after updating config to username/password."""
         return await self.async_step_user()
+
+    @staticmethod
+    @callback
+    def async_get_options_flow(
+        config_entry: ConfigEntry,
+    ) -> OptionsFlow:
+        """Create the options flow."""
+        return HydrawiseOptionsFlow(config_entry)
+
+
+class HydrawiseOptionsFlow(OptionsFlow):
+    """Handle a Hydrawise options flow."""
+
+    def __init__(self, config_entry: ConfigEntry) -> None:
+        """Initialize options flow."""
+        self.config_entry = config_entry
+
+    async def async_step_init(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
+        """Manage the options."""
+        if user_input is not None:
+            return self.async_create_entry(title="", data=user_input)
+
+        options = self.config_entry.options
+        return self.async_show_form(
+            step_id="init",
+            data_schema=vol.Schema(
+                {
+                    vol.Optional(
+                        CONF_SCAN_INTERVAL,
+                        default=options.get(CONF_SCAN_INTERVAL, DEFAULT_SCAN_INTERVAL),
+                    ): int
```

Comment:
Hi there @thomaskistler 👋 

I'm going to friendly decline this pull request, as we do not accept a configurable scan interval.

Instead, we should adjust the default to be sane and usable for everyone; users that want to adjust the scan interval beyond that sane default, can use the `homeassistant.update_entity` service in an automation to define their own schedules.

../Frenck